docs/user_guide.md

```diff
@@ -719,6 +720,76 @@ If you are a project admin, you can disable a Robot Account by clicking "Disable
 If you are a project admin, you can delete a Robot Account by clicking "Delete" in the `Robot Accounts` tab of a project.
 ![delete_robot_account](img/robotaccount/disable_delete_robot_account.png)
 
+## Configure Webhook Notifications
+
+If you are a project admin, you can configure a connection from a project in Harbor to a webhook endpoint. If you configure webhooks, Harbor notifies the webhook endpoint of certain events that occur in the project. Webhooks allow you to integrate Harbor with other tools to streamline continuous integration and development processes. 
+
+The action that is taken upon receiving a notification from a Harbor project depends on your continuous integration and development process. For example, by configuring Harbor to send a `POST` request to a webhook listener at an endpoint of your choice, you can trigger a build and deployment of an application whenever there is a change to an image in the repository.
+
+### Supported Events
+
+You can define one webhook per project. Webhook notifications provide information about events in JSON format and are delivered by `HTTP` or `HTTPS POST` to an existing webhhook endpoint URL that you provide. The following table describes the events that trigger notifications and the contents of each notification.

```

Comment:
You can configure multiple webhooks per project, but only one endpoint

Reply:
I added that you can add multiple webhooks per project, but I cannot see how to do so in the UI. @reasonerjt ?

Reply:
alex means that you can define one webhook policy to subscribe multiple webhook events.
one webhook policy per project. one endpoint  for policy but multiple events for it.

Reply:
OK, that's what I now have in the doc:

"You can define one webhook endpoint per project. Webhook notifications provide information about events in JSON format and are delivered by `HTTP` or `HTTPS POST` to an existing webhhook endpoint URL that you provide."